src/main/scala/scorex/waves/http/DebugApiRoute.scala

```diff
@@ -31,7 +31,7 @@ import DebugApiRoute._
 
 
 @Path("/debug")
-@Api(value = "/debug")
+@Api(value = "/_debug")
```

Comment:
why need to change the value here?

Reply:
For console error output in the issue
![image](https://user-images.githubusercontent.com/39894573/55147531-8675aa00-5181-11e9-9d0e-18c68e4080dd.png)